### DISCUSSION TITLE
posts/jekyll-chirpy-text-and-typography/

### DISCUSSION THREAD

**Discussion by giscus** [lamngockhuong/dev#1] [General]
# posts/jekyll-chirpy-text-and-typography/

Titles H1 - heading

https://dev.ngockhuong.com/posts/jekyll-chirpy-text-and-typography/

**Comment by jjnevis** [comment]
Hi there! Jekyll/Chirpy noob here, just wondering if you know how I can remove the "header" from the rendered code block (with the three dots, </> text and copy button?
Thanks!
